scripts/build_tdx.sh

```
@@ -108,15 +108,14 @@
     cp -fv qemu/sp-qemu-tdx*.deb package/
     cp -fv edk2/Build/OvmfX64/DEBUG_GCC5/FV/OVMF.fd package/
     cd package
-    tar -czvf ../package.tar.gz .
+    tar -czvf ../package-tdx.tar.gz .
     popd
```

Shellcheck (0.10.0)

[warning] Use 'popd ... || exit' or 'popd ... || return' in case popd fails.

(SC2164)